packages/main/src/plugin/tasks/progress-impl.ts

```diff
@@ -78,14 +83,34 @@ export class ProgressImpl {
     );
   }
 
+  protected getTaskAction(options: extensionApi.ProgressOptions): TaskAction | undefined {
+    if (!options.navigation) return undefined;
+
+    if (!this.navigationManager.hasRoute(options.navigation.routeId)) {
+      console.warn(`cannot created task action for unknown routeId ${options.navigation.routeId}`);
+      return undefined;
+    }
+
+    return {
+      name: 'Goto task >',
+      execute: (): unknown => {
+        if (!options.navigation) return;
+        return this.navigationManager.navigateToRoute(options.navigation.routeId, ...options.navigation.arguments);
```

Comment:
Could it be possible (and ok) to close the Task window at this point?

Reply:
We currently do not have a way to close the TaskManager from the backend, so might be a follow-up issue